### PR TITLE
fix: Context menu close should restore focus

### PR DESCRIPTION
**Pull request by ling1726**
Fixes regression from #28530

The `contextMenuProps` aren't really for just the context menu but common props for all menus fter the
refactor in #25044. We should probably revisit the menu trigger code and clean it up but this fix is more urgent

### CODE REVIEW

File: packages/react-components/react-menu/stories/Menu/MenuDefault.stories.tsx
Comment (bsunderhus) [inline packages/react-components/react-menu/stories/Menu/MenuDefault.stories.tsx:1]: Should the default story be the one about context?

**Review by bsunderhus — APPROVED** [review]

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/InfoButton-mount_base.html">6</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/InfoButton-mount.html">11</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/InfoButton-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/Avatar-mount_base.html">632</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/Avatar-mount.html">632</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/Button-mount_base.html">304</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/Button-mount.html">298</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/Field-mount_base.html">1088</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/Field-mount.html">1117</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/FluentProvider-mount_base.html">676</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/FluentProvider-mount.html">711</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/FluentProviderWithTheme-mount_base.html">81</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/FluentProviderWithTheme-mount.html">80</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">63</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">64</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">79</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">87</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/InfoButton-mount_base.html">6</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/InfoButton-mount.html">11</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/InfoButton-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/MakeStyles-mount_base.html">892</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/MakeStyles-mount.html">861</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/Persona-mount_base.html">1767</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/Persona-mount.html">1713</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/SpinButton-mount_base.html">1388</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29290/perf-test-react-components/SpinButton-mount.html">1347</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                                                 |      Baseline (minified/GZIP) |                            PR |                                                                                                Change |
| :-------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: | ----------------------------: | ----------------------------------------------------------------------------------------------------: |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>              | `140.184 kB`<br />`43.176 kB` | `140.184 kB`<br />`43.178 kB` | <br />`2 B`  |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr> |  `142.92 kB`<br />`43.714 kB` |  `142.92 kB`<br />`43.715 kB` | <br />`1 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `69.572 kB`<br />`19.658 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `208.688 kB`<br />`59.552 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `40.966 kB`<br />`13.569 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `6.541 kB`<br />`2.227 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/b11ab1c1968c8317cb6262b6c5f51cf6d41662fe'>b11ab1c1968c8317cb6262b6c5f51cf6d41662fe</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Pull request merged**